octobot_trading/api/trader.py

```diff
@@ -0,0 +1,59 @@
+#  Drakkar-Software OctoBot-Trading
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+from octobot_trading.api import LOGGER_TAG
+from octobot_commons.logging.logging_util import get_logger
+
+LOGGER = get_logger(LOGGER_TAG)
+
+
+def is_trader_enabled_in_config(exchange_manager):
```

Comment:
Can you add some typing ?
`def is_trader_enabled_in_config(exchange_manager) -> bool`
`def is_trader_enabled(exchange_manager) -> bool`

etc...